pkg/tsdb/testdatasource/scenarios.go

```diff
@@ -370,6 +377,21 @@ func (s *Service) handleRandomWalkWithErrorScenario(ctx context.Context, req *ba
 	return resp, nil
 }
 
+func (s *Service) handleDataResponseErrorStatusScenario(_ context.Context, req *backend.QueryDataRequest) (*backend.QueryDataResponse, error) {
+	resp := backend.NewQueryDataResponse()
+
+	rand.Seed(time.Now().Unix())
```

Comment:
can we set the seed to be based on the query time range?  This makes it possible to have a stable response we can use in testing.  But when using a "to=now" it is random behavior